packages/@uppy/utils/src/RateLimitedQueue.js

```diff
@@ -198,6 +211,9 @@ class RateLimitedQueue {
     }
     this.#downLimit = this.limit
     this.limit = Math.ceil((this.#upperLimit + this.#downLimit) / 2)
+    for (let i = this.#downLimit; i <= this.limit; i++) {
+      this.#queueNext()
+    }
```

Comment:
nit: might be nice to write a tiny `times(n, fn)` function for these kinds of behaviors.